handout/tooling/chrome.md

```diff
@@ -1,6 +1,6 @@
-## Chrome
+## Web Browser
 
 We use Google's Chrome browser for this course because of its cutting-edge JavaScript engine and excellent debugging tools.
 
-However, code written with AngularJS should work on any modern web browser (Firefox, IE9+, Chrome, Safari).
+Some who champion the freedom of the 'Open Free Internet' use Mozilla Firefox, there is a [Developer Edition](https://www.mozilla.org/en-US/firefox/developer/) available with development and debugging tools. However, code written with JavaScript should work on any modern web browser (Firefox, IE9+, Chrome, Safari, Opera).
```

Comment:
Ah ya, this could turn into a political hot topic, was trying to keep rangle.io un-biased. Students see us as leaders and will just go with what we say, suggest. Also FF lives and dies off it's install base, just keeping things open and fair.